index.d.ts

```diff
@@ -67,6 +67,7 @@ export interface HeaderButtonsProps {
    * However, you're free to use your own component (see `HeaderButton` for reference).
    */
   HeaderButtonComponent?: ComponentType<any>;
+  children: ReactChild | Array<ReactNode>;
```

Comment:
seems to me that just `ReactNode` should be enough, no? ReactNode already includes array of children from what I can tell

Reply:
yes, I tested with `ReactNode` it is working fine, if you want I can push the new commit